package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "dotenv": "^8.1.0",
-    "jQuery": "^1.7.4",
+    "jquery": "^3.4.1",
```

Comment:
npm gave a warning to change the package to jquery, all lowercase, so I updated it per suggestion

Reply:
Didn't we get rid of jquery from the other templates? I don't see it in office-addin-taskpane-js.

Reply:
It is being used, but the code could be modified to not use it. For example, here's the usage in word.ts:
```
$(document).ready(function() {
      $("#getGraphDataButton").click(ssoAuthHelper.getGraphData);
```

which could be changed to use `document` and  `document.getElementById("sideload-getGraphDataButton")`

Reply:
I will try that out and see if it works. If so, I will remove jquery

Reply:
That does indeed work and obviates the need for jquery.  Pushing up changes soon